test/integration/TokenTransfers.spec.ts

```diff
@@ -59,55 +58,111 @@ describe('Token transfers', () => {
                 identity1.account,
                 TBD_URI,
                 5,
-                )
-        }).to.throw()   
+            )
+        }).to.throw()
     })
 
-    it('should throw an error when trying to send negative amount', function() {
+    it('should throw an error when trying to send negative amount', function () {
         expect(() => {
             RadixTransactionBuilder.createTransferAtom(
                 identity1.account,
                 account2,
                 TBD_URI,
                 -1,
-                )
-        }).to.throw()   
+            )
+        }).to.throw()
     })
 
-    it('should throw an error when trying to send zero', function() {
+    it('should throw an error when trying to send zero', function () {
         expect(() => {
             RadixTransactionBuilder.createTransferAtom(
                 identity1.account,
                 account2,
                 TBD_URI,
                 0,
-                )
-        }).to.throw()   
+            )
+        }).to.throw()
     })
 
-    it('should throw an error when trying to send too many tokens', function() {
+    it('should throw an error when trying to send too many tokens', function () {
         expect(() => {
             RadixTransactionBuilder.createTransferAtom(
                 identity1.account,
                 account2,
                 TBD_URI,
                 5000000,
-                )
-        }).to.throw()   
+            )
+        }).to.throw()
+    })
+
+    it('should mint and transfer tokens', async function() {
+        this.timeout(50000)
+
+        const symbol = 'TBA'
+        const name = 'my token name'
+        const description = 'my token description'
+        const granularity = 0.01
+        const amount = 500
+        const iconUrl = 'http://a.b.com'
+        const TBA_URI = `/${identity1.account.getAddress()}/TBA`
+
+        const createToken = new Promise((resolve, reject) => {
+            new RadixTransactionBuilder().createTokenMultiIssuance(
+                identity1.account,
+                name,
+                symbol,
+                description,
+                granularity,
+                amount,
+                iconUrl,
+            )
+                .signAndSubmit(identity1)
+                .subscribe({
+                    complete: () => resolve(),
+                    error: e => reject(new Error(JSON.stringify(e))),
+                })
+
+        })
+
+        await createToken
+
+        expect(identity1.account.transferSystem.tokenUnitsBalance[TBA_URI].toString()).to.eq('500')
+        expect(account2.transferSystem.tokenUnitsBalance[TBA_URI]).to.be.undefined
+
+        const mintAndTransfer = new Promise((resolve, reject) => {
+            new RadixTransactionBuilder().mintAndTransferTokens(
+                identity1.account,
+                TBA_URI,
+                4000,
+                identity1.account,
+                account2,
+                "message",
```

Comment:
Single quotes please

Reply:
I think we need to add a Linter to the project 😃. It's on my todo list.